### DISCUSSION TITLE
Separate customization of in-app and email notifications

### DISCUSSION THREAD

**Discussion by amoralesc** [outline/outline#6045] [Ideas]
Currently, there is no way to customize in-app notifications if the email integration is disabled. What this means is that an user with site-wide access will receive in-app notifications to multiple events with no means to disable them, unless they also have an SMTP server. While this isn't ground-breaking, it would be nice to have an option for an uncluttered notification tray.

## Current state with email integration disabled
![image](https://github.com/outline/outline/assets/78283504/987732fb-6707-4d5c-9e2c-7ab7c187a205)

## Proposed solution
![image](https://github.com/outline/outline/assets/78283504/1471ff97-560c-4684-9d4a-35b995597bd0)

**Comment by tommoor** [comment]
Feel free to create an issue for this
